lib/commands/deleteUser.ts

```diff
@@ -0,0 +1,26 @@
+import { FetchResult } from "apollo-link";
+import gql from "graphql-tag";
+
+import { fetchManagement } from "../../src/utils/fetchManagement";
```

Comment:
The fact that `lib` calls a function from `account` (i.e. `fetchManagement`) is a bad design from my part, as we plan to migrate `lib` into its own package. I just realised that we have this in a lot of part of `lib`. I think this is out of the scope of this PR.

Reply:
Do you want me to make a ClickUp task to remember that we need to do something about that ?

Reply:
Yes please 🙇.